variables.mk

```diff
@@ -88,6 +88,7 @@ KIND_WAIT := 30s
 ### ci vars
 # timeout for short test suite, must be parsable as Go time.Duration (60m, 2h)
 SHORT_CI_TIMEOUT := 60m
+SANITY_TEST := [Testpattern: Dynamic PV (default fs)] volumes should store data
```

Comment:
Can we use `?=` instead of `:=` ?

Reply:
Idk, actually I just declared this variable the same as others are declared. What's the advantage of your proposal?

Reply:
https://ftp.gnu.org/old-gnu/Manuals/make-3.79.1/html_chapter/make_6.html#SEC64

> If you'd like a variable to be set to a value only if it's not already set,

Reply:
`SANITY_TEST` will be used for running target locally `make SANITY_TEST='some focus' ...` while `SHORT_CI_TIMEOUT` mostly in CI pipelines.